pkg/option/config.go

```diff
@@ -1175,6 +1175,17 @@ const (
 	// running and able to schedule endpoints.
 	WriteCNIConfigurationWhenReady = "write-cni-conf-when-ready"
 
+	// DefaultCNIConfiguration is an original default CNI configuration file. If
```

Comment:
I had a bit of a think, and I think we can get away with a single string option. Something called like "CNIChainNetwork". If it's set, then:

1. Wait for a CNI network with that name to exist
2. Read that, generate the chained configuration (either from default or from read-cni-configuration)
3. Write the generated config to disk

What do you think?